code/datums/mood_events/generic_positive_events.dm

```diff
@@ -11,6 +11,13 @@
 /datum/mood_event/betterhug/add_effects(mob/friend)
 	description = "<span class='nicegreen'>[friend.name] was very nice to me.</span>\n"
 
+//NSV 13 - caress the torp.
+/datum/mood_event/torphug
+	description = "<span class='nicegreen'>I really needed that..</span>\n"
+	mood_change = 1
+	timeout = 2 MINUTES
+//NSV13 end
+
```

Comment:
```suggestion
```
Ditto,
Since all of this is unique to NSV we can just define it in /nsv13/code/datums/mood_events/nsv_events.dm instead

Reply:
Why did it place this comment second... :tragedy: